packages/gatsby-theme-carbon/src/components/AccordionGroup.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from '@emotion/styled'
+import { settings } from 'carbon-components'
+
+const { prefix } = settings
+
+const AccordionGroupHeader = styled.h3`
```

Comment:
We want to avoid mixing css-in-js with sass modules

Reply:
Replace css-in-js with sass module.